cdap-ui/app/features/dashboard/templates/widgets/table.html

```diff
@@ -6,7 +6,12 @@
     </thead>
     <tbody>
     <tr ng-repeat="row in tableData">
-      <td ng-repeat="value in row track by $index">{{::value}}</td>
+      <td ng-repeat="value in row track by $index">
+        <!--dddd, MMMM Do YYYY, h:mm:ss a-->
+        <div ng-if="$first">{{::value * 1000 | amDateFormat:'M/D/YY h:mm:ss a'}}</div>
```

Comment:
It's one time binding: https://docs.angularjs.org/guide/expression#one-time-binding